cypress/helpers/cypressTasks/cognito/cognito-helpers.ts

```diff
@@ -180,3 +213,59 @@ export const deleteAllCypressTestAccounts = async (): Promise<null> => {
   await Promise.all(accounts.map(user => deleteAccount(user, userPoolId)));
   return null;
 };
+
+export const deleteAllIrsCypressTestAccounts = async (): Promise<null> => {
+  const irsUserPoolId = await getUserPoolId(true);
+  if (!irsUserPoolId) return null;
+  const irsAccounts = await getAllCypressTestAccounts(irsUserPoolId);
+  await Promise.all(
+    irsAccounts.map(user => deleteAccount(user, irsUserPoolId)),
+  );
+  return null;
+};
+
+export async function getBearerToken({
```

Comment:
This function logs errors specific to the IRS Superuser, that's fine but then it doesn't make sense to me to have `isIrsEnv` be a variable since I am not sure the function is set up to handle or is even relevant to other user types.